### DISCUSSION TITLE
Wrap left axis and make it bold

### DISCUSSION THREAD

**Discussion by melarenina** [plouc/nivo#1597] [Q&A] [answered]
Hey :)

is there any way to wrap the left axis, so it doesn't get too long? and also, any way that I can make it bold?

Many thanks!

**Comment by wyze** [comment, marked as answer]
You can use a custom `renderTick` function and make the elements bold and try something like `tspans` to wrap your text. We do not have anything built in for wrapping long labels.

**Reply by melarenina** [reply]
How about for bar charts? It seems like it does not have this property.

**Reply by wyze** [reply]
Sorry you pass an object to axisLeft prop of the bar chart. One of the properties of the axisLeft prop is renderTick.
